examples/angular/apps/angular/src/app/shared/data.service.spec.ts

```diff
@@ -0,0 +1,29 @@
+/* tslint:disable:no-unused-variable */
+
+import { TestBed, async, inject } from '@angular/core/testing';
+import { DataService } from './data.service';
+import { SubService } from './sub.service';
+
+const getTitleFn = jest.fn()
+  .mockReturnValue('Test');
+const subServiceSpy = jest.fn()
+  .mockImplementation(() => ({
+    getTitle: getTitleFn
+  }))
+
+describe('Service: DataService', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        DataService,
+        // { provide: SubService, useValue: subServiceSpy() } this works
```

Comment:
we can plug in a working version now, and as part of the bug fix (whenever it lands) we can add a `useClass` example

Reply:
done